imports/util/cloudinary-transformations.js

```diff
@@ -1,4 +1,4 @@
-const generalReplacer = (url, paramsStr) => url.replace(/\/upload\/[^/]*\//, `/upload/${paramsStr}/`)
+const generalReplacer = (url, paramsStr) => url.replace(/\/upload\/[^/]*\//, `/upload/f_auto/${paramsStr}/`)
```

Comment:
That won't work for most image rendering places, because we are using the `paramsStr` to convert the images to a smaller appropriate size to the space they occupy

Reply:
I don't see them conflicting. Tbh I don't quite understand how the code works, so I just took a stab in this instance.

Reply:
> Tbh I don't quite understand how the code works

AFAICT, by looking at cloudinary's documentation, all the conversion params need to be comma separated between one set of slashes ( / )

Reply:
Ah, so should `f_auto` be added to matchWidth & fitDimensions?